src/modules/settings/index.js

```diff
@@ -1,151 +1,13 @@
+/* eslint-disable react/jsx-filename-extension */
```

Comment:
if a file imports react it should be `.jsx`

Reply:
This will interfere with the import glob logic